isthmus/src/test/java/io/substrait/isthmus/ArithmeticFunctionTest.java

```diff
@@ -156,4 +156,50 @@ void acosh(String column) throws Exception {
     String query = String.format("SELECT ACOSH(%s) FROM numbers", column);
     assertFullRoundTrip(query, CREATES);
   }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"i8", "i16", "i32", "i64"})
+  void bitwise_not_scalar(String column) throws Exception {
+    String query = String.format("SELECT BITNOT(%s) FROM numbers", column);
+    assertFullRoundTrip(query, CREATES);
+  }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"i8, 8", "i16, 160", "i32, 32000", "i64, CAST(6000000004 AS BIGINT)"})
```

Comment:
You could use a [CsvSource](https://docs.junit.org/current/user-guide/#writing-tests-parameterized-tests-sources-CsvSource) instead of a ValueSource to pass multiple parameters into the test and avoid parsing a single string yourself.

Reply:
It's such a small change, I've just gone ahead and pushed it to your PR, as well as rebasing on the latest repo content. I'll merge as soon as the build confirms the tests pass.